src/renderer/components/table/table-cell.tsx

```diff
@@ -15,6 +16,10 @@ export interface TableCellProps extends React.DOMAttributes<HTMLDivElement> {
   isChecked?: boolean; // mark checkbox as checked or not
   renderBoolean?: boolean; // show "true" or "false" for all of the children elements are "typeof boolean"
   sortBy?: TableSortBy; // column name, must be same as key in sortable object <Table sortable={}/>
+  style?: any;
```

Comment:
This should not be overridden, `React.DOMAttributes<T>` has a perfectly good typing for this field.

Reply:
Do you mean [HTMLAttributes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0893371fea43bfdf1777b6d835424961ba0d1dbb/types/react/index.d.ts#L1765)? DOMAttributes doesn't contain this prop

Reply:
Ah okay, then please change this to `React.CSSProperties`